test-case/check-audio-equalizer.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+##
+## Case Name: check-audio-equalizer.sh
+## Preconditions:
+##    SOF Topology should have EQ component(s) in any pipeline
+## Description:
+##    check IIR/FIR config working
+## Case step:
+##    1. Check if the topology has a EQ component in it
+##    2. Test with IIR config list
+##    3. Test with FIR config list
+## Expect result:
+##    sof-ctl and aplay should return 0
+##
+
+# source from the relative path of current folder
+source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+
+OPT_OPT_lst['t']='tplg'     OPT_DESC_lst['t']='tplg file, default value is env TPLG: $TPLG'
+OPT_PARM_lst['t']=1         OPT_VALUE_lst['t']="$TPLG"
+
+OPT_OPT_lst['d']='duration' OPT_DESC_lst['d']='aplay duration in second'
+OPT_PARM_lst['d']=1         OPT_VALUE_lst['d']=5
+
+OPT_OPT_lst['l']='loop'     OPT_DESC_lst['l']='loop count'
+OPT_PARM_lst['l']=1         OPT_VALUE_lst['l']=1
+
+OPT_OPT_lst['s']='sof-logger'   OPT_DESC_lst['s']="Open sof-logger trace the data will store at $LOG_ROOT"
+OPT_PARM_lst['s']=0             OPT_VALUE_lst['s']=1
+
+func_opt_parse_option $*
+tplg=${OPT_VALUE_lst['t']}
+duration=${OPT_VALUE_lst['d']}
+loop_cnt=${OPT_VALUE_lst['l']}
+
+func_pipeline_export $tplg "type:playback"
```

Comment:
@aiChaoSONG @Bin-QA Can we direct export tplg with EQ here?

Reply:
@xiulipan EQ test need eq tplg, this is not absolutely required. I can work with @aiChaoSONG @Bin-QA as an enhancement. if you don't have any other comment, please approve this.

Reply:
OK, can you add TODO and add issue to track the enhancement.
As you said, we need special TPLG for the test. So check if the TPLG support EQ as early as we can is needed.

Reply:
@xiulipan @fredoh9 our current pipeline filter function in sof-tplgreader.py can only cover pipeline type filtering, this is not good, I will try to refine it, and add more filtering feature.